pkg/database/dto/user.go

```diff
@@ -88,6 +88,7 @@ type FacilityStaffResponse struct {
 	Web        []*UserResponse `json:"web" yaml:"web" xml:"web"`
 	Instructor []*UserResponse `json:"instructor" yaml:"instructor" xml:"instructor"`
 	Mentor     []*UserResponse `json:"mentor" yaml:"mentor" xml:"mentor"`
+	MIT        []*UserResponse `json:"mit" yaml:"mit" xml: "mit"`
```

Comment:
Use MentorInTraining(?) vs MIT if possible here.. keep acronyms to those defined by VATUSA to help alleviate confusion